### PR TITLE
fix deprecated translation

### PR DESCRIPTION
**Pull request by buptliuwei**
fix deprecated translation

### CODE REVIEW

File: content/zh/docs/concepts/policy/pod-security-policy.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ weight: 30
 <!--
 PodSecurityPolicy is deprecated as of Kubernetes v1.21, and will be removed in v1.25.
 -->
-PodSecurityPolicy 在 Kubernetes v1.21 版本中被启用，将在 v1.25 中删除。
+PodSecurityPolicy 在 Kubernetes v1.21 版本中被弃用，将在 v1.25 中删除。
```
Comment (SataQiu) [inline content/zh/docs/concepts/policy/pod-security-policy.md:20]: Good catch! @buptliuwei 
But it seems that this page has fallen behind the English version, can you update the whole page?
You can check if a page is in sync with the English version with `./scripts/lsync.sh`.
For example：
```sh
./scripts/lsync.sh content/zh/docs/concepts/policy/pod-security-policy.md
diff --git a/content/en/docs/concepts/policy/pod-security-policy.md b/content/en/docs/concepts/policy/pod-security-policy.md
index f0884c3de..fac2b1205 100644
--- a/content/en/docs/concepts/policy/pod-security-policy.md
+++ b/content/en/docs/concepts/policy/pod-security-policy.md
@@ -464,12 +464,12 @@ allowed prefix, and a `readOnly` field indicating it must be mounted read-only.
 For example:
 
 ```yaml
-allowedHostPaths:
-  # This allows "/foo", "/foo/", "/foo/bar" etc., but
-  # disallows "/fool", "/etc/foo" etc.
-  # "/foo/../" is never valid.
-  - pathPrefix: "/foo"
-    readOnly: true # only allow read-only mounts
+  allowedHostPaths:
+    # This allows "/foo", "/foo/", "/foo/bar" etc., but
+    # disallows "/fool", "/etc/foo" etc.
+    # "/foo/../" is never valid.
+    - pathPrefix: "/foo"
+      readOnly: true # only allow read-only mounts
 ```
 
 {{< warning >}}There are many ways a container with unrestricted access to the host
```

Comment (buptliuwei) [reply content/zh/docs/concepts/policy/pod-security-policy.md:20]: ok i will do it

**Comment by chenxuc** [comment]
Thanks for the good catch.
/lgtm

**Pull request merged**